### PR TITLE
Better examples for tracing

### PR DESCRIPTION
**Pull request by pksunkara**
- chore: Update rust-cli references to clap-rs

Since we moved the repo to `clap-rs` org, I went and updated the references.

- docs(examples): Update examples

I have updated the tracing example to be much more simpler.

### CODE REVIEW

**Review by matthiasbeyer — APPROVED** [review]

**Comment by matthiasbeyer** [comment]
Thanks! No need to do this "conventional commit" stuff though! :smile:

**Comment by pksunkara** [comment]
Because we moved it to `clap-rs` org. I have merge permissions too but will wait for epage to take a look.

File: Cargo.toml [hunk 1]
```diff
@@ -52,4 +52,4 @@ clap = { version = "4.0.0", default-features = false, features = ["std", "derive
 env_logger = "0.10.0"
 tracing = "0.1"
 tracing-subscriber = "0.3"
-tracing-log = "0.1.3"
+tracing-log = "0.1"
```
Comment (epage) [inline Cargo.toml:55]: nit: Why was this changed?

I've been slowly been making my version requirements more specific

File: examples/tracing.rs [hunk 2]
```diff
@@ -0,0 +1,24 @@
+use clap::Parser;
```
Comment (epage) [inline examples/tracing.rs:1]: Side note: In the future, try to do renames in separate commits so other changes are easier to review

File: Cargo.toml (hunk 1 above)
Comment (pksunkara) [reply Cargo.toml:55]: That was a mistake. I thought I made them specific, but it was the reverse lol.
